docs/user-guide/configuration-settings.md

```diff
@@ -28,6 +28,8 @@ The configuration keys are hierarchical. This structure is most convenient to ma
 |  | AllowInsecureHttp | `false` | Controls how the OpenID Connect server (ASOS) handles the incoming requests to arriving on non-HTTPS endpoints should be rejected or not. By default, this property is set to false to help mitigate man-in-the-middle attacks.
 |  | Hangfire          | E.g. <br>"Hangfire": {<br>            "JobStorageType": "Memory",<br>            "UseHangfireServer": true,<br>            "AutomaticRetryCount": 1,<br>"WorkerCount": 11,<br>"Queues": [<br>"alpha",<br>"default"<br>],<br>            "SqlServerStorageOptions": {<br>                "CommandBatchMaxTimeout": "00:05:00",<br>                "SlidingInvisibilityTimeout": "00:05:00",<br>                "QueuePollInterval": "00:00:00",<br>                "UseRecommendedIsolationLevel": true,<br>                "UsePageLocksOnDequeue": true,<br>                "DisableGlobalLocks": true,<br>                "EnableHeavyMigrations": true<br>            }<br>        }   | Background processing library (Hangfire) configuration.<br><br> `JobStorageType` - current job storage. Supported values: Memory, SqlServer.<br> `UseHangfireServer` - enable/disable HangFire for this app instance.<br> `AutomaticRetryCount` - maximum number of automatic retry attempts. <br> `WorkerCount` - Hangfire subsystem dedicated pool of worker threads size.<br> `Queues` - Set of the queues Hangfire uses for prioritizing jobs.<br> `SqlServerStorageOptions` - Hangfire.SqlServer.SqlServerStorageOptions. Check [Hangfire's SQL Server Configuration](https://docs.hangfire.io/en/latest/configuration/using-sql-server.html#configuration) for details.
 |  | Swagger | E.g. <br>"Swagger":<br>{<br>"Disable": true<br>}| Allows to disable swagger initialization at platform startup to prevent access to the swagger UI and documents.
+|  | UseAllOfToExtendReferenceSchemas | `true` | Extend Rest API reference schemas (using the allOf construct) so that contextual metadata.
```

Comment:
```suggestion
|  | UseAllOfToExtendReferenceSchemas | `true` | Extend Rest API reference schemas (using the allOf construct) so that contextual metadata can be applied to all parameter and property schemas.
```